src/app/_components/Table/TableHeaderWithTooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@headlessui/react'
+import { Question } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
+
+import { ClickTooltip, type ClickTooltipProps } from '@/components/ClickTooltip'
+import { Icon } from '@/components/Icon'
+
+type TableHeaderWithTooltipProps = {
+  header: string
+  description: ClickTooltipProps['description']
+}
+
+const touchTarget = {
+  iconSize: 20,
+  iconWrapperPadding: 'p-1.5',
+  class: 'p-2',
+  offsetClass: '-m-2',
+}
```

Comment:
I believe this is the pattern I've been seeing lately.

```suggestion
const TOUCH_TARGET = {
  iconSize: 20,
  iconWrapperPadding: 'p-1.5',
  class: 'p-2',
  offsetClass: '-m-2',
} as const
```

Reply:
✅